docs/api-diffs/9.0/rc.1/diff.md

```diff
@@ -0,0 +1,1290 @@
++            public static IResourceBuilder<IConstructResource<Bucket>> AddObjectCreatedNotification(this IResourceBuilder<IConstructResource<Bucket>> builder, IResourceBuilder<IConstructResource<ITopic>> destination, params INotificationKeyFilter[] filters);
++            public static IResourceBuilder<IConstructResource<Bucket>> AddObjectCreatedNotification(this IResourceBuilder<IConstructResource<Bucket>> builder, IResourceBuilder<IConstructResource<IQueue>> destination, params INotificationKeyFilter[] filters);
++            public static IResourceBuilder<IConstructResource<Bucket>> AddObjectRemovedNotification(this IResourceBuilder<IConstructResource<Bucket>> builder, IResourceBuilder<IConstructResource<ITopic>> destination, params INotificationKeyFilter[] filters);
++            public static IResourceBuilder<IConstructResource<Bucket>> AddObjectRemovedNotification(this IResourceBuilder<IConstructResource<Bucket>> builder, IResourceBuilder<IConstructResource<IQueue>> destination, params INotificationKeyFilter[] filters);
++            public static IResourceBuilder<IConstructResource<Bucket>> AddS3Bucket(this IResourceBuilder<IStackResource> builder, [ResourceNameAttribute] string name, IBucketProps? props = null);
++            public static IResourceBuilder<TDestination> WithReference<TDestination>(this IResourceBuilder<TDestination> builder, IResourceBuilder<IConstructResource<Bucket>> bucket, string? configSection = null) where TDestination : IResourceWithEnvironment;
++        }
++        public static class SNSResourceExtensions {
++            public static IResourceBuilder<IConstructResource<Topic>> AddSNSTopic(this IResourceBuilder<IStackResource> builder, [ResourceNameAttribute] string name, ITopicProps? props = null);
++            public static IResourceBuilder<IConstructResource<Topic>> AddSubscription(this IResourceBuilder<IConstructResource<Topic>> builder, IResourceBuilder<IConstructResource<IQueue>> destination, SqsSubscriptionProps? props = null);
++            public static IResourceBuilder<TDestination> WithReference<TDestination>(this IResourceBuilder<TDestination> builder, IResourceBuilder<IConstructResource<Topic>> topic, string? configSection = null) where TDestination : IResourceWithEnvironment;
++        }
++        public static class SQSResourceExtensions {
++            public static IResourceBuilder<IConstructResource<Queue>> AddSQSQueue(this IResourceBuilder<IStackResource> builder, [ResourceNameAttribute] string name, IQueueProps? props = null);
++            public static IResourceBuilder<TDestination> WithReference<TDestination>(this IResourceBuilder<TDestination> builder, IResourceBuilder<IConstructResource<Queue>> queue, string? configSection = null) where TDestination : IResourceWithEnvironment;
++        }
+     }
+     namespace Aspire.Hosting.AWS {
+-        public class AWSProvisioningException : Exception {
```

Comment:
breaking change

Reply:
I see it was moved to a new namespace:  namespace Aspire.Hosting.AWS.Provisioning {
+        public class AWSProvisioningException : Exception {
+            public AWSProvisioningException(string message, Exception? innerException = null);
+        }
+    }